src/Mailer/Email.php

```diff
@@ -1925,6 +1929,25 @@ protected function _renderTemplates($content)
             $rendered[$type] = rtrim($rendered[$type], "\n");
         }
 
+        /* Embed images inline in html templates */
+        if (!empty($rendered['html'])) {
+            $rendered['html'] = str_replace(['file:', 'file://', 'cid://'], 'cid:', $rendered['html']);
+            $hasEmbedImages = preg_match_all('~(["\'])cid:([^\1]+)\1~iU', $rendered['html'], $embedImages);
+            if ($hasEmbedImages > 0) {
+                $embedImages = array_unique($embedImages[2]);
+                foreach ($embedImages as $file) {
+                    if (is_file($file)) {
```

Comment:
This is a potential security hole. If you open up a form to the public and let the write the text of the email, they can basically send themselves any file in the system.